### PR TITLE
Add header files to cordb{di, ee_dac, ee_wks}.vcxproj to improve developer experience

### PR DESCRIPTION
**Pull request by 4creators**
Work toward #14884

### CODE REVIEW

**Comment by 4creators** [comment]
cc @janvorli @jkotas PTAL

**Comment by 4creators** [comment]
CI package restore failures

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked Innerloop Build and Test (Jit - TieredCompilation=0)
@dotnet-bot test Windows_NT x86 Checked Innerloop Build and Test (Jit - TieredCompilation=0)

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked Innerloop Build and Test no_tiered_compilation
@dotnet-bot test Windows_NT x86 Checked Innerloop Build and Test no_tiered_compilation

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked no_tiered_compilation
@dotnet-bot test Windows_NT x86 Checked no_tiered_compilation

**Comment by 4creators** [comment]
Single test failure for Ubuntu x64 Checked Build and Test (Jit - TieredCompilation=0)  
JIT_Regression._VS_ia64_JIT_M00_b80373_b80373_b80373_._VS_ia64_JIT_M00_b80373_b80373_b80373_sh [FAIL] 
https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x64_checked_ubuntu_no_tiered_compilation_flow_prtest/5/

**Comment by 4creators** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test (Jit - TieredCompilation=0)

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked no_tiered_compilation

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bet test Ubuntu arm Cross Checked no_tiered_compilation_innerloop Build and Test
@dotnet-bot test Ubuntu x64 Checked Build and Test
@dotnet-bot test Ubuntu x64 Checked Build and Test (Jit - TieredCompilation=0)
@dotnet-bot test Ubuntu x64 Checked CoreFX Tests 
@dotnet-bot test Ubuntu x64 Checked Innerloop Build and Test 
@dotnet-bot test Ubuntu x64 Formatting
@dotnet-bot test Windows_NT x64 Checked Innerloop Build and Test (Jit - TieredCompilation=0) 
@dotnet-bot test Windows_NT x64 Formatting

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked no_tiered_compilation Build and Test

**Comment by 4creators** [comment]
@dotnet-bot test Ubuntu x64 Checked no_tiered_compilation Build and Test

**Comment by janvorli** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test (Jit -  TieredCompilation=0)

File: src/debug/di/CMakeLists.txt [hunk 1]
```diff
@@ -34,6 +34,21 @@ set(CORDBDI_SOURCES
   valuehome.cpp
 )
 
+if (WIN32)
```
Comment (janvorli) [inline src/debug/di/CMakeLists.txt]: Can you please set the CORDBI_HEADERS unconditionally?

**Comment by 4creators** [comment]
@janvorli Addressed code review. Failing tests are caused but IO errors.

**Pull request merged**